src/core/modules/listeners/listeners_manager.h

```diff
@@ -33,6 +33,11 @@
 #include "utilities/wrap_macros.h"
 #include "utlvector.h"
 
+// This is required for accessing m_nFlags without patching convar.h
+#define private public
+#include "convar.h"
+#undef private
```

Comment:
I don't think you need that private/public hack here since you don't access any private symbols which should not produce an error when the generated object file is processed during the compilation. That said, it could probably be best to have an `utilities/convars.h` that does it and include from there instead (similar to how we have an `utilities/baseentity.h` that ensure everything is included in order, etc.) so that it is consistent across the board.

Reply:
> I don't think you need that private/public hack here since you don't access any private symbols which should not produce an error when the generated object file is processed during the compilation.

You're right, changed it.

> That said, it could probably be best to have an `utilities/convars.h` that does it and include from there instead (similar to how we have an `utilities/baseentity.h` that ensure everything is included in order, etc.) so that it is consistent across the board.

That's a good idea. I will test it in another branch and PR it.